index.js

```diff
@@ -190,56 +190,97 @@ class ServerlessCustomDomain {
   }
 
   /*
+   * Look through list of certs and see if any match the one we are looking for
+   */
+  findClosetCertificate(data) {
+    // The more specific name will be the longest
+    let nameLength = 0;
+    // The arn of the choosen certificate
+    let certificateArn;
+    // The certificate name
```

Comment:
These comments aren't very useful. Looks like this has been here for a while but would be a good time to clean it up while this code is being changed